app/components/form/closed-question.js

```diff
@@ -5,6 +5,10 @@ import OpenQuestionComponent from './open-question';
 export default OpenQuestionComponent.extend({
   store: service(),
   questionTypes: ClosedQuestionTypes,
+  init() {
+    this._super();
+    this.send('addOption');
```

Comment:
What does this do? This is only run once, not every time this component is rendered, so I am not sure if this does what it is supposed to do

Reply:
this is done such that the closed question type always starts with one dummy/example option.